keps/sig-instrumentation/1602-structured-logging/README.md

```diff
@@ -373,28 +373,26 @@ Klog interface was selected as it is already supported by klog.v2 with `SetLogge
 
 Introduce structured logging and JSON format:
 * Most important logs are migrated to structured methods.
-* Flag for selecting logging format is implemented
-* JSON format is implemented
+* Flag for selecting logging format is implemented.
+* JSON format is implemented.
 
 #### Beta
 
-Adding guarding against regression and update user guideline:
-* Static analysis protects important log lines from reverting to string formating
-* [Kubernetes Logging convention](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md) document is updated
-
-Components in scope:
-* Kubelet
-
-All other component migrations will be best effort.
+Bring feature quality up to and prevent regression during ongoing migration:
+* Directories can be marked as migrated, preventing regression via verify script in `hack`.
+* [Kubernetes Logging documentation](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md) is updated for structured logging.
+* Json format should support same set of feature flags as text format, minus those we will decide to deprecte.
+* Structured logging can be used for [logging performance traces](https://github.com/kubernetes/utils/tree/master/trace) or alternative is proposed.
```

Comment:
Why is this a graduation criteria? If the feature is available, this can be used; this should only be tracked if we plan on doing a migration as part of this KEP, and IMO this is not in scope, that should be tracked separately (and it looks to be out-of-tree anyways).

Reply:
sg